project/build.scala

```diff
@@ -21,6 +21,7 @@ object Version {
   lazy val akkaVersion    = "2.4.16"
   lazy val akkaHttpCorsVersion = "0.2.1"
   lazy val scalaLoggingVersion = "3.7.2"
+  lazy val sparkCoreVersion = "2.1.1"
```

Comment:
We can also just update Spark altogether here but it seemed worth it to keep the old version around for SummaryJob etc.